mypy.ini

```diff
@@ -0,0 +1,9 @@
+# mypy.ini
+[mypy]
+check_untyped_defs = True
+disallow_untyped_defs = False
+ignore_missing_imports = True
```

Comment:
Do we need `ignore_missing_imports` option? Mainly asking this out of curiosity on what dependency is causing issues.